portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// photos data
+import photos_data from './photos-data.js';
+
 /**
- * Adds a random greeting to the page.
+ * Create photo component with caption 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function createPhoto(photo) {
+    let component = [
+        '<div class="col-4">',
```

Comment:
[Use template literals for complex string concatenation](https://google.github.io/styleguide/jsguide.html#features-strings-template-strings)

So this function could be something more like:
```js
function renderPhoto({src, location, date}) {
  return `
    <img src="${src}" alt="${location}">
  `;
}
```